examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>you are probably planning that already, but the <code>1</code>  and <code>2</code>  should be references: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#references</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Now I am confused, is ok to use markdown links for example to point to Wikipedia or other webpages? Or we should always use latex/bibtex style? For example here `1` and `2` are links to web pages not scientific papers. `Poisson processes` and `Cox processes` are links to Wikipedia articles

Reply:
Wikipedia links and to other doc pages are fine, yes. I commented because I opened one and looked at it for a whole 2 seconds and got the impression it was a vignette that would then have a citation with the package. No need to do it if it's not the case